recipes-core/busybox/busybox_1.33.%.bbappend

```diff
@@ -1,50 +1,51 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}:${THISDIR}/files:${THISDIR}/${PN}:"
+FILESEXTRAPATHS:prepend := "${THISDIR}:${THISDIR}/files:${THISDIR}/${PN}:"
 
 SRC_URI =+ " \
-            file://busybox-ifplugd \
-            file://ifplugd.action \
-            file://ifplugd.conf \
-            file://zcip.script \
-            file://busybox-acpid \
-            file://acpid.conf \
-            file://acpid_poweroff.sh \
-            file://acpid-logrotate.conf \
-            file://zcip-allow-action-script-to-reject-chosen-IP.patch \
-            file://login-utilities.cfg \
-            file://udhcpd.wlan0.conf"
+			file://busybox-ifplugd \
+			file://ifplugd.action \
+			file://ifplugd.conf \
+			file://zcip.script \
+			file://busybox-acpid \
+			file://acpid.conf \
+			file://acpid_poweroff.sh \
+			file://acpid-logrotate.conf \
+			file://zcip-allow-action-script-to-reject-chosen-IP.patch \
+			file://login-utilities.cfg \
+			file://udhcpd.wlan0.conf"
 
-SRC_URI_append_x64 += "file://enable_ar_create_fragment.cfg"
+SRC_URI:append:x64 += "file://enable_ar_create_fragment.cfg"
 
 # we're using syslog-ng, don't build busybox syslog (avoids build warnings)
-SRC_URI_remove += "file://syslog.cfg"
-INITSCRIPT_PACKAGES_remove += "${PN}-syslog"
+SRC_URI:remove += "file://syslog.cfg"
+INITSCRIPT_PACKAGES:remove += "${PN}-syslog"
 
 # Do not perform update-rc.d actions on the hwclock.sh initscript in this
 # package. We only wish to call hwclock.sh from /etc/init.d/bootmisc manually.
-INITSCRIPT_PACKAGES_remove = "${PN}-hwclock"
+INITSCRIPT_PACKAGES:remove = "${PN}-hwclock"
 
 PACKAGES =+ " ${PN}-ifplugd"
 PACKAGES =+ " ${PN}-acpid"
 PACKAGES =+ " ${PN}-zcip"
 
 DEPENDS =+ " libselinux"
 
-FILES_${PN}-ifplugd = "${sysconfdir}/init.d/busybox-ifplugd ${sysconfdir}/ifplugd/ifplugd.action ${sysconfdir}/ifplugd/ifplugd.conf"
-FILES_${PN}-acpid = "${sysconfdir}/init.d/busybox-acpid ${sysconfdir}/acpid.conf ${sysconfdir}/acpi ${sysconfdir}/acpi/poweroff.sh"
-FILES_${PN}-zcip = "${sysconfdir}/natinst/networking/zcip.script"
-FILES_${PN}-udhcpd =+ "${sysconfdir}/udhcpd.wlan0.conf"
+FILES:${PN}-ifplugd = "${sysconfdir}/init.d/busybox-ifplugd ${sysconfdir}/ifplugd/ifplugd.action ${sysconfdir}/ifplugd/ifplugd.conf"
+FILES:${PN}-acpid = "${sysconfdir}/init.d/busybox-acpid ${sysconfdir}/acpid.conf ${sysconfdir}/acpi ${sysconfdir}/acpi/poweroff.sh"
+FILES:${PN}-zcip = "${sysconfdir}/natinst/networking/zcip.script"
+FILES:${PN}-udhcpd =+ "${sysconfdir}/udhcpd.wlan0.conf"
 
 INITSCRIPT_PACKAGES =+ " ${PN}-acpid"
 
-INITSCRIPT_NAME_${PN}-acpid = "busybox-acpid"
-INITSCRIPT_PARAMS_${PN}-acpid = "start 20 2 3 4 5 . stop 20 0 1 6 ."
+INITSCRIPT_NAME:${PN}-acpid = "busybox-acpid"
+INITSCRIPT_PARAMS:${PN}-acpid = "start 20 2 3 4 5 . stop 20 0 1 6 ."
 
 # Remove default busybox udhcpd init script; on NILRT images
 # udhcpd is invoked directly from ifplugd action scripts
+# unset line must be left as old syntax for now or it's unparsed.
 unset INITSCRIPT_NAME_${PN}-udhcpd
```

Comment:
Calling this out for specific review. I'm not sure why this didn't work with the new syntax.

Reply:
That's interesting. I wonder if upstream forgot to accommodate the new syntax in the `unset` operation. :thinking:

Reply:
Either way, I'm fine with leaving it in the old syntax until we get to the kirkstone rebase.

Reply:
Created https://dev.azure.com/ni/DevCentral/_workitems/edit/2114544, will link the work item number here